watcher/tool/watcher/alerts.go

```diff
@@ -115,7 +127,9 @@ func receiverLoop(ctx context.Context, kubeClient *kubeapi.Clientset, kClient *k
 	}
 }
 
-func createAlert(client *kapacitor.Client, spec []byte, log *log.Entry) error {
+func createAlert(rClient resources.Resources, spec []byte, log *log.Entry) error {
```

Comment:
nit: is there a real need to prefix the client  variable? I think `client` was good enough unless you want to focus the reader on the fact that it not a simple client (and she'll be puzzled just what `r` means)? Or is there ambiguity with `client`?